discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
+            "apiDocsUrl": "http://www.rubydoc.info/gems/discordrb",
+            "tutorials": "yes",
+            "tutorialsUrl": "https://github.com/meew0/discordrb/wiki",
+            "examples": "yes",
+            "examplesUrl": "https://github.com/meew0/discordrb/tree/master/examples",
+            "eventsAsync": "threaded",
+            "apiCallsAsync": "thread-safe",
+            "commandExtension": "yes",
```

Comment:
Whether this lib has features that allow easy creation of command-based bots. There were no issues with this last time; are there any other features your lib has that could be added here?